coldfront/plugins/qumulo/views/update_allocation_view.py

```diff
@@ -120,10 +181,19 @@ def form_valid(
         for key in access_keys:
             access_users = form_data[key + "_users"]
             self.set_access_users(key, access_users, allocation)
-        
+
         # needed for redirect logic to work
         self.success_id = str(allocation.id)
 
+    def form_valid(
```

Comment:
Repostingn for visibility, since it disappeared with file-update:

Should be moved higher up on the page for readability. When I'm looking over a view, I'm looking for this method because I know django is calling it during the submission flow.